Firestore/Source/Util/FSTDispatchQueue.h

```diff
@@ -42,7 +42,11 @@ typedef NS_ENUM(NSInteger, FSTTimerID) {
    * A timer used in FSTOnlineStateTracker to transition from OnlineState Unknown to Offline
    * after a set timeout, rather than waiting indefinitely for success or failure.
    */
-  FSTTimerIDOnlineStateTimeout
+  FSTTimerIDOnlineStateTimeout,
+  /**
+   * A timer used to periodically attempt LRU Garbage collection
+   */
+  FSTLruGCTimer
```

Comment:
A few comments on this name:
  * Objective-C enums need to start with the type name, so this should be FSTTimerID...
  * Does this specifically need to be about LRU GC? If not, how about about "garbage collection"
  * Is this an idle time? An interval? A delay?

Taken together this could be FSTTimerIDGarbageCollectionDelay or similar.